src/api/api.ts

```diff
@@ -37,6 +40,9 @@ export type UbrukteTilgangInvitasjoner = z.infer<typeof UbrukteTilgangInvitasjon
 export type UbesluttetTilgangForesporsel = z.infer<typeof UbesluttetTilgangForesporselSchema>
 export type UbesluttedeTilgangForesporsler = z.infer<typeof UbesluttedeTilgangForesporslerSchema>
 
+export type EndringsmeldingType = z.infer<typeof EndringsmeldingSchema>
+export type EndringsmeldingerType = z.infer<typeof EndringsmeldingerSchema>
```

Comment:
Trenger vi `Type`-suffix ?, isåfall så burde det være konsekvent i resten av filen